R/error-helpers.R

```diff
@@ -412,7 +412,7 @@ abort_is_not_dm <- function(obj_class) {
 }
 
 error_is_not_dm <- function(obj_class) {
-  glue("Required class `dm` but instead is {tick(obj_class)}")
+  glue("Required class `dm` but instead is {paste(tick(obj_class), collapse = ', ')}")
```

Comment:
```suggestion
  glue("Required class `dm` but instead is {format_classes(obj_class)}")
```

with a suitable implementation.